lesson-one/examples/hello-world-dom.html

```diff
@@ -0,0 +1,30 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="dom.css"/>
+		<title>Dom's Space Merchandise</title>
+	</head>
+	<body>
+		<h1>Dom's Space Merchandise</h1>
+		<table border=1>
+			<tr>
+				<td>
+					Grab a
+				</td>
+				<td>
+					seat
+				</td>
+			</tr>
+			<tr>
+				<td colspan=2>
+					at the table!
+				</td>
+			</tr>
+		</table>
+		<p>
+			<!-- <a href="#">this is a link</a> -->
+		</p>
+		<p>
+			<img src="planet.png" width=400/>
```

Comment:
Again, better to declare the width in your css file, rather than as an attribute